tools/validators/instance_validator/validate/point.py

```diff
@@ -12,22 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Parses Telemetry Payload"""
-
-from __future__ import print_function
+"""Data point in a telemetry message."""
 
 class Point(object):
   """A point from a pubsub telemetry message.
 
   Args:
-    message: a pubsub message
+    point_name: name of the point
+    value: value of the point
 
   Returns:
```

Comment:
is it the best practice in python to have a returns on the class? I dont know

Reply:
Hm, probably not. I've removed this section from the docstrings of all the classes I added + Point.